Tests/Tracking/APITest.swift

```diff
@@ -135,7 +135,7 @@ class TrackingAPITest: UnitTest {
             "version": sdkWrapperVersion
         ]
 
-        var actual = CioSdkConfig.Factory.create(region: .US)
+        var actual = CioSdkConfig.Factory.create(siteId: "", apiKey: "", region: .US)
```

Comment:
can we use an internal convenience initializer for testing only, instead of providing empty values every time?